internal/custom-code-logging/command_custom_code_logging_create.go

```diff
@@ -0,0 +1,110 @@
+package customcodelogging
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	cclv1 "github.com/confluentinc/ccloud-sdk-go-v2/ccl/v1"
+
+	pcmd "github.com/confluentinc/cli/v4/pkg/cmd"
+	"github.com/confluentinc/cli/v4/pkg/examples"
+	"github.com/confluentinc/cli/v4/pkg/output"
+)
+
+func (c *customCodeLoggingCommand) newCreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create a custom code logging.",
+		Args:  cobra.ExactArgs(0),
+		RunE:  c.createCustomCodeLogging,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Create custom code logging.`,
+				Code: "confluent custom-code-logging create --cloud aws --region us-west-2 --environment env-000000 --topic topic-123 --cluster-id cluster-123",
+			},
+		),
+	}
+
+	pcmd.AddCloudFlag(cmd)
+	pcmd.AddRegionFlagKafka(cmd, c.AuthenticatedCLICommand)
+	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
+	cmd.Flags().String("topic", "", "Kafka topic of custom code logging destination.")
+	cmd.Flags().String("cluster-id", "", "Kafka cluster id of custom code logging destination.")
+	cmd.Flags().String("log-level", "", "Log level of custom code logging. (default \"INFO\").")
+
+	pcmd.AddContextFlag(cmd, c.CLICommand)
+	pcmd.AddOutputFlag(cmd)
+
+	cobra.CheckErr(cmd.MarkFlagRequired("cloud"))
+	cobra.CheckErr(cmd.MarkFlagRequired("region"))
+	cobra.CheckErr(cmd.MarkFlagRequired("environment"))
+	cmd.MarkFlagsOneRequired("topic")
+	cmd.MarkFlagsOneRequired("cluster-id")
```

Comment:
@sgagniere since this is Kafka cluster ID, maybe we can make the `cluster` as part of the context, so if user doesn't specify cluster ID, then it will assume to use the current cluster ID? Just as the case to produce/consume.